### ISSUE TITLE
Make preCICE config file used for solverdummy follow the tutorials naming convention

### ISSUE THREAD

**Issue by IshaanDesai** [precice/python-bindings#172]
Similar to https://github.com/precice/precice/issues/1454

The precice-config file: https://github.com/precice/python-bindings/blob/develop/examples/solverdummy/precice-config.xml does not follow the naming conventions set in https://precice.org/community-contribute-to-precice.html#naming-conventions.

**Comment by phershbe** [comment]
Hello Ishaan, I'd like to work on this if possible. It doesn't seem too hard, and the guidance from the other pull request helps. Thank you.

**Comment by IshaanDesai** [comment]
@phershbe sure, please fork the repo and open a pull request to the develop branch when you have done the changes. As you have correctly noted, the changes are straightforward and the pull request mentioned above is a very good reference.

**Comment by phershbe** [comment]
Thank you, I'm going to work on it today. I was a little bit behind because I've been traveling. Where I'm at is that I've downloaded the dependencies on Homebrew and I think next is to do the build. I also noticed that the previous pull request had had some difficulties in that part. It was going fine for me but then I was busy with other stuff. I'll get it in later today or send an update.

**Comment by phershbe** [comment]
Hello @IshaanDesai, an update ... I got behind on this, I'm getting things going though. I think I've cleared up some of my confusion now because the other example mentioned, https://github.com/precice/precice/pull/1548, is regarding https://github.com/precice/precice and this is regarding https://github.com/precice/python-bindings, correct? (Not your fault, as I'm new I got confused though.) So I followed https://precice.org/installation-source-preparation.html and got everything built and the tests passing, but then I realized that I was in the wrong repository.

Now I need to follow https://precice.org/installation-bindings-python.html I think. So I'm going to work on this later today. I think for somebody new to the project and this specific issue, building and testing everything is most of the task and updating the names and making sure the tests correspond should be relatively fast. Let me know if there is anything that I understand incorrectly above or if you have anything to add. Thank you.

**Comment by IshaanDesai** [comment]
> think I've cleared up some of my confusion now because the other example mentioned, https://github.com/precice/precice/pull/1548, is regarding https://github.com/precice/precice and this is regarding https://github.com/precice/python-bindings, correct?

Correct. `precice/precice` is the repository of the coupling library preCICE, and `precice/python-bindings` is the repository of the Python bindings of the preCICE API.

To run and test things locally, you need to install preCICE itself, and then the Python bindings on top of it. So you have actually followed the procedure correctly.
